exercises/leap/leap_test.go

```diff
@@ -2,17 +2,23 @@ package leap
 
 import "testing"
 
-// Define a function with the following signature:
-// IsLeapYear(int) bool
-
+// targetTestVersion is used to ensure that a solution is only evaluated
+// against the correct version of the test suite.
 const targetTestVersion = 3
 
+// TestTestVersion compares the targetTestVersion defined above with
+// a testVersion value.
+// This is a common convention throughout the Go Exercism track.
+// To make a test like this test pass, define 'testVersion' in your solution.
+// We've done this for you in the ./leap.go file provided.
 func TestTestVersion(t *testing.T) {
 	if testVersion != targetTestVersion {
 		t.Fatalf("Found testVersion = %v, want %v", testVersion, targetTestVersion)
 	}
 }
 
+// Define a function with the following signature:
+// IsLeapYear(int) bool
```

Comment:
I'm not sure this is necessary as the function signature is already defined in the stub file. https://github.com/exercism/go/blob/master/exercises/leap/leap.go#L5

And, in fact, I think the  [comment within the function](https://github.com/exercism/go/blob/master/exercises/leap/leap.go#L6) could probably be got rid of as it is introduced in `two-fer`.